Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1901,8 +1975,8 @@ internal extension Tensor where Scalar: TensorFlowFloatingPoint {
     @inlinable
     func _vjpSum(squeezingAxes axes: Tensor<Int32>) -> (Tensor, (Tensor) -> Tensor) {
         let value = sum(squeezingAxes: axes)
-        return (value, { [shape = shapeTensor] v in
-	      let unsqueezed = v.expandingShape(at: axes.scalars.map { Int($0) })
+        return (value, { [shape = shapeTensor] seed in
```

Comment:
I'd suggest using "v" instead of "seed". We've been avoiding the word seed in both the AD implementation and the library.

Reply:
I addressed the comments, but I'm curious, why don't we use a more descriptive name?

Reply:
You could call it "tangent" or "direction", but "seed" is only confusing because only the really traditional automatic differentiation community calls it "seed".

Reply:
I see...yeah a more descriptive name would be better but the best I can think of at the moment is `backpropagatedGradient` which is too long. Let's leave it for now I guess. :)